src/studio-header/HeaderBody.jsx

```diff
@@ -165,6 +170,7 @@ HeaderBody.propTypes = {
   outlineLink: PropTypes.string,
   searchButtonAction: PropTypes.func,
   containerProps: PropTypes.shape(Container.propTypes),
+  onLogoNavigate: PropTypes.func.isRequired,
```

Comment:
```suggestion
```

I don't think this is used anymore, and it's causing a warning in the JS console.

Reply:
Removed, nice catch!